src/main/java/org/apache/commons/mail/util/MimeMessageParser.java

```diff
@@ -262,7 +262,7 @@ private boolean isMimeType(final MimePart part, final String mimeType)
      * @param part   the current part to be processed
      * @return the DataSource
      * @throws MessagingException creating the DataSource failed
-     * @throws IOException        creating the DataSource failed
+     * @throws IOException        error getting InputStream or unsupported encoding
```

Comment:
If you throw specific IOExceptions like UnsupportedEncodingException, document it too.